workers/common/src/main/kotlin/common/env/EnvironmentService.kt

```diff
@@ -101,7 +101,7 @@ class EnvironmentService(
         repositoryService: InfrastructureService?
```

Comment:
Commit message nit: This is not a change of the `dao` module; `workers` would be more appropriate, I think. Same for the next commit.